x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/user/RestPutUserAction.java

```diff
@@ -58,10 +58,13 @@ public RestChannelConsumer innerPrepareRequest(RestRequest request, NodeClient c
         return channel -> requestBuilder.execute(new RestBuilderListener<PutUserResponse>(channel) {
             @Override
             public RestResponse buildResponse(PutUserResponse putUserResponse, XContentBuilder builder) throws Exception {
-                return new BytesRestResponse(RestStatus.OK,
-                        builder.startObject()
-                                .field("user", putUserResponse)
-                                .endObject());
+                builder.startObject()
+                    .startObject("user"); // TODO in 7.0 remove wrapping of response in the user object and just return the object
```

Comment:
@jaymode now that this change landed, there's nothing holding us back removing this artifact, right?

Reply:
that is correct

Reply:
I opened #33512